hw1/02-palindrome.js

```diff
@@ -1,3 +1,28 @@
 const elem = document.querySelector('input');
+const resultDiv = document.querySelector('#result');
+const palindromeText = document.createElement('p');
+palindromeText.textContent = '';
+resultDiv.appendChild(palindromeText);
```

Comment:
I think the div is supposed to increase slightly when you add the text. The only minor difference I see between your solution and the HW screenshot is that there is a little more space between your palindrome analysis text and the bottom of the container box. I think getting rid of the <p> element on line 3 would fix that. This seemed to work: 

const elem = document.querySelector("input");
const palindromeText = document.querySelector("#result");
palindromeText.textContent = "";